### PR TITLE
Expose quicklisp location to user

### PR DESCRIPTION
**Pull request by cab-1729**
Allow for different quicklisp location.

### CODE REVIEW

File: repl.lisp [hunk 1]
```diff
@@ -1,5 +1,22 @@
 #!/usr/bin/env -S sbcl --script
-(load "~/quicklisp/setup")
+
+(defvar *repl-version* "0.1.4")
+(defvar *repl-name*    "Veit's REPL for SBCL")
+(defvar *prompt*       "sbcl> ")
+(defvar *prompt2*       "....> ")
+(defvar *ret*          "=> ")
+(defvar *quicklisp*    "~/quicklisp/setup")
```
Comment (vindarel) [inline repl.lisp:8]: Isn't it setup.lisp ? (with extension)

**Comment by vindarel** [comment]
There might be an issue if we wanted to quickload some libraries in our init file right?

**Comment by cab-1729** [comment]
> There might be an issue if we wanted to quickload some libraries in our init file right?

init file? I don't get it. Please forgive me, I'm a little new to Lisp.

**Comment by vindarel** [comment]
>  I'm a little new to Lisp.

it's great you send PRs already :)

I should have said sbcli's config file.

we load one:

```lisp
(if (probe-file *config-file*)
  (load *config-file*))
```

with your PR we load quicklisp afterwards.

What if we want to quickload a library in our config file, before quicklisp is loaded?

(there might be an issue, I don't have the answer)

**Comment by cab-1729** [comment]
If the user intends to to use ```quickload``` in the ```sbclirc``` shouldn't he have to load quicklisp in the same file separately?
If the user does so then there would be no issue.
It makes sense for a user to load dependencies in the same file they are being used, rather than relying on it being already loaded in another dependency. I think this is a general programming practice.
For example, this is the reason why we use ```#ifndef``` header guards in C. To stop a dependency from being loaded more than once, because the programmer is expected to load each dependency in each file it is used even if those files are compiled together. I hope I'm making sense.

**Pull request closed without merging**